editor.planx.uk/src/@planx/graph/index.ts

```diff
@@ -408,12 +408,7 @@ const _update = (
 
       // eslint-disable-next-line no-prototype-builtins
       const isRemoved = !isSomething(v) && existingData.hasOwnProperty(k);
-      const isUpdatedArray =
-        Array.isArray(v) && !isEqual(v, (existingData as any)[k]);
-      const isUpdatedPrimitive =
-        typeof v !== "object" && v !== (existingData as any)[k];
-      const isUpdated =
-        isSomething(v) && (isUpdatedPrimitive || isUpdatedArray);
+      const isUpdated = isSomething(v) && !isEqual(v, (existingData as any)[k]);
```

Comment:
This is a non-functional change, just a tidy up I spotted when looking for the cause of the issue.

`isEqual()` can compare anything, so we don't need to worry about separating out primitives and arrays.